cylc/flow/cfgspec/workflow.py

```diff
@@ -115,19 +160,20 @@
             Configure the directories and files to be included in the remote
             file installation.
 
-             .. note::
-                These, as standard, include the following directories:
+            .. note::
+
+               The following directories are installed by default:
 
-                 * app
-                 * bin
-                 * etc
-                 * lib
+                * app
+                * bin
+                * etc
+                * lib
 
-                 And include the server.key file (from the .service
-                 directory), this is required for authentication.
+                And include the server.key file (from the .service
+                directory), this is required for authentication.
 
-            These should be located in the top level of your Cylc workflow,
-            i.e. the directory that contains your flow.cylc file.
+               These should be located in the top level of your Cylc workflow,
+               i.e. the directory that contains your flow.cylc file.
```

Comment:
I think this indentation is wrong, it needs an extra space to make it a part of the `.. note` or two fewer spaces to make it part of the text?